torch/fx/graph.py

```diff
@@ -1801,6 +1803,18 @@ def has_side_effect(node):
                 self.erase_node(node)
                 changed = True
 
+        # Call DCE on the subgraphs
+        if self.owning_module is not None:
+            subgraph_names: OrderedSet[str] = OrderedSet()
+            for node in sorted(self.find_nodes(op="get_attr")):
```

Comment:
any reason for the sorting? Are you worried that if we run DCE on inner subgraphs in a non-deterministic order, that this will affect the final artifact in some way?

Reply:
No real reason, i can get rid of it.

Reply:
Done